aiutils/tftools/images.py

```diff
@@ -0,0 +1,22 @@
+import tensorflow as tf
+
+def resize_images_like(images, images_like, method=0, align_corners=False):
```

Comment:
Argument for "like" is https://www.tensorflow.org/versions/r0.8/api_docs/python/constant_op.html#zeros_like

How about reference_tensor?